### DISCUSSION TITLE
Two json! macros?

### DISCUSSION THREAD

**Discussion by jafioti** [rwf2/Rocket#1785] [Questions]
I'm sorry if this is obvious, but I didn't see it anywhere in the docs yet, and I got messed up for a few hours cause of it. Is there a reason that there is a `rocket::serde::json::json!` macro and a `rocket::serde::json::serde_json::json!` macro? 

I have been trying to use them the same as the original `serde_json::json!` macro from the serde_json crate, but the first one (`rocket::serde::json::json!`) kept giving me an unresolved macro error. I eventually stumbled upon the rocket::serde::json::serde_json::json!` macro, and it works as expected.

Is there a reason for two macros that are named the same and seemingly do the same thing, except the better named one doesn't work? Sorry if I skimmed over something in the changelog.

**Comment by SergioBenitez** [comment]
There is no `rocket::serde::json::serde_json::json!`, only `rocket::serde::json::json!`, which as evidenced in the docs and examples works as expected.
